### ISSUE TITLE
suggestion: optional mp title change when mp recovered

### ISSUE THREAD

**Issue by androidwiltron** [teltek/Galicaster#519]
mediapackage/repository.py
`recover_recording` sets a prefix to the title of the mediapackage. if the mediapackage came from opencast, really you should maintain a 1:1 copy of the metadata.
I suggest this to be either optional or for scheduled mp turned off

```
            mp.setTitle("Recovered - " + mp.getTitle())
```

**Comment by smarquard** [comment]
I agree it would be preferable to keep the title unchanged, but at the moment this is the only indication in the media manager UI that the recording didn't complete as it was supposed to (other than ingest not done).

Perhaps an additional column with a warning exclamation mark or something would help.

**Comment by androidwiltron** [comment]
this would do it:
```
            if mp.manual:
                mp.setTitle("Recovered - " + mp.getTitle())
```

but now ive found an exception in when saving manual mp's :laughing: ... typical!
i'll investigate further but for me at least:
manual mp. kill gc and restart you get an exception when recovering the mp.
turns out the deserializer is looking for a '0' rather than a 'None' here in the manifest.xml
`<mediapackage duration="None"`
